src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2634,6 +2637,10 @@ private void makeEmailStateConsistent(FeedbackSessionAttributes oldSession,
     private void sendFeedbackSessionPublishedEmail(FeedbackSessionAttributes session) {
         addFeedbackSessionReminderToEmailsQueue(session, EmailType.FEEDBACK_PUBLISHED);
     }
+    
+    private void sendFeedbackSessionUnpublishedEmail(FeedbackSessionAttributes session) {
+        addFeedbackSessionReminderToEmailsQueue(session, EmailType.FEEDBACK_UNPUBLISHED);
```

Comment:
Let's add a test for this too, there are similar tests in `FeedbackSessionEmailTaskQueueTest`